tests/profile/test_basic_suite_builder_profiler.py

```diff
@@ -22,127 +20,14 @@
 from great_expectations.render.renderer_configuration import MetaNotesFormat
 from great_expectations.self_check.util import (
     expectationSuiteValidationResultSchema,
-    get_dataset,
 )
-from great_expectations.util import is_library_loadable
 
 if TYPE_CHECKING:
     from great_expectations.data_context import AbstractDataContext
 
 FALSEY_VALUES = [None, [], False]
 
 
-@pytest.fixture
-def datetime_dataset(test_backend):
```

Comment:
I've deleted this fixture and the tests that use it since they are marked as xfail with the reason that `get_dataset` should no longer be called. This fixture sets up data via that call.

Reply:
Thanks for cleaning this up @billdirks